package/package.mk

```diff
@@ -13,11 +13,11 @@ $(crossplane_bin): | $(go_bin)
 
 # Install up plugin
 $(up_bin):export GOBIN = $(go_bin)
-$(up_bin):export VERSION=v0.15.0
+$(up_bin):export VERSION=v0.30.0
```

Comment:
Is the up binary still necessary? Afaik the crossplane crank binary can do all the operations as well. At least we only use crank in the main appcat repo.

In fact we use these operations already here, it would be nice if we could cut down the dependencies a bit.

Also, instead of installing up/crank like we do now we could do it the same way we do in the main appcat repository:

Import it to a `tools.go` file and then use `go run github.com/crossplane/crossplane/cmd/crank`. This has the advantage that the versions of the tools are automatically managed by `go.mod`.

Reply:
Thanks for the tip. I removed up all together and use crossplane binary.